bigbluebutton-html5/client/main.html

```diff
@@ -83,6 +83,17 @@
   <script src="compatibility/adapter.js?v=VERSION" language="javascript"></script>
   <script src="compatibility/sip.js?v=VERSION" language="javascript"></script>
   <script src="compatibility/kurento-utils.js?v=VERSION" language="javascript"></script>
+  <!-- fonts -->
+  <link rel="preload" href="fonts/BbbIcons/bbb-icons.woff?j1ntjp" as="font" crossorigin="anonymous"/>
+  <link rel="preload" href="fonts/SourceSansPro/SourceSansPro-Light.woff" as="font" crossorigin="anonymous"/>
```

Comment:
Could we do a wildcard `*` loading from the fonts dir?
This way people could add fonts without having to explicitly preload.
This is just a question, I have not checked whether it's good idea or not

Reply:
no, it's not possible, the files must be explicitly listed